### PR TITLE
Feature/fa 1057 configurable batches

### PR DESCRIPTION
**Pull request by CH-JonathanBlake**
https://companieshouse.atlassian.net/browse/FA-1057

Removed hard coded query batch sizes and replaced with variable than is configured with env settings.

### CODE REVIEW

File: server/config/.env [hunk 1]
```diff
@@ -32,4 +32,6 @@ export APPLICATIONS_DEVELOPER_SERVICE_FUTURE_BASE_URL=${APPLICATIONS_API_FUTURE_
 export APPLICATIONS_DEVELOPER_SERVICE_FUTURE_API_KEY=
 export APPLICATIONS_DEVELOPER_SERVICE_FUTURE_USERNAME=
 export APPLICATIONS_DEVELOPER_SERVICE_FUTURE_PASSWORD=
+
```
Comment (ICraig-CH) [inline server/config/.env:35]: sorry I think this extra line was my fault

Comment (CH-JonathanBlake) [reply server/config/.env:35]: Its not an unreasonable way to split up the variables, happy to leave it if you think its more readable, or remove it. No preferences either way.

Comment (ICraig-CH) [reply server/config/.env:35]: I don't mind , I suppose it does make it more readable so I guess there is no harm in leaving it

**Review by ICraig-CH — APPROVED** [review]
Happy to approve, hopefully nobody makes over a million applications (we could always change it then) :)

**Review by ch-rfishwick — APPROVED** [review]

**Review by Zain-Abbas1 — APPROVED** [review]

**Pull request merged**